src/main/java/longtimenosee/logic/commands/PersonCommand.java

```diff
@@ -7,7 +7,7 @@
  */
 public class PersonCommand extends Command {
 
-    public static final String COMMAND_WORD = "showClient";
+    public static final String COMMAND_WORD = "clients";
```

Comment:
Shall we change _clients_ to _contacts_ too for standardization? What are your thoughts?